stock_lock_lot/models/stock_quant.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2015 Serv. Tec. Avanzados - Pedro M. Baeza (http://www.serviciosbaeza.com)
+# Copyright 2015 AvanzOsc (http://www.avanzosc.es)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class StockQuant(models.Model):
+    _inherit = "stock.quant"
+
+    locked = fields.Boolean(
```

Comment:
I think this is not adding any value and takes time to computed (imagine on installing the module...). Or is it used in any place?

Reply:
It is only used to show in the tree an form view if its product is locked, so i can remove it.